core/controller.go

```diff
@@ -251,6 +295,15 @@ func (c *Controller) StopStreamHandler(w http.ResponseWriter, r *http.Request, p
 		c.sendError(w, err, http.StatusInternalServerError)
 		return
 	}
+
+	if dto.ID == "" && len(dto.Alias) > 0 {
```

Comment:
It would be nice to check here if both of them are empty. Because at that point the request is invalid anyway.

```go
if dto.ID == "" && len(dto.Alias) == 0 {
  // throw back not found
}
if len(dto.Alias) > 0 {
  // set dto.ID
}
// proceed further
```

Reply:
Thank you @Roverr. This covid-19 is definitely changing daily normal around the planet. 

I have made the code changes.